website/content/docs/cloudinary.md

```diff
@@ -30,55 +30,101 @@ media_library:
 The following options are specific to the Netlify CMS integration for Cloudinary:
 
 * **`output_filename_only`**: _(default: `false`)_\
-  By default, the value provided for a selected image is a complete URL for the asset on Cloudinary's CDN. Setting `output_filename_only` to `true` will instead produce just the filename (e.g. `image.jpg`).
+  By default, the value provided for a selected image is a complete URL for the asset on Cloudinary's CDN. Setting `output_filename_only` to `true` will instead produce just the filename (e.g. `image.jpg`). This should be `true` if you will be directly embedding cloudinary transformation urls in page tempaltes. Refer to [Embedding Cloudinary url in templates] ()
```

Comment:
End of this sentence needs to be completed (missing url, space between brackets and parens, needs a period).

Reply:
Fixed